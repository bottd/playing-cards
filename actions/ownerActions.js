export const actionTypes = {
  resetOwner: 'RESET_OWNER',
  setOwner: 'SET_OWNER',
}

export const resetOwner = () => ({
  type: actionTypes.resetOwner,
});

export const setOwner = () => ({
  type: actionTypes.resetOwner,
});
