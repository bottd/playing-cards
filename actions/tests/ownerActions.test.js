import { actionTypes, resetOwner, setOwner } from '../ownerActions';

describe('Owner actions', () => {
  describe('action types', () => {
    it('Should match snapshot', () => {
      expect(actionTypes).toMatchSnapshot();
    });
  });

  describe('resetOwner', () => {
    it('Should return an action with correct type', () => {
      const expected = { type: 'RESET_OWNER' };
      const result = resetOwner();
      expect(expected).toEqual(result);
    });
  });

  describe('setOwner', () => {
    it('Should return an action with correct type', () => {
      const expected = { type: 'SET_OWNER' };
      const result = setOwner();
      expect(expected).toEqual(result);
    });
  });
});
