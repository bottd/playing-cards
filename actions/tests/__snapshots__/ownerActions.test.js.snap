// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Owner actions action types Should match snapshot 1`] = `
Object {
  "resetOwner": "RESET_OWNER",
  "setOwner": "SET_OWNER",
}
`;
